### DISCUSSION TITLE
生成pdf之后，pdf文件上丢失小节的书签

### DISCUSSION THREAD

**Discussion by during123** [note286/xduts#167] [Q&A] [answered]
模版建好之后是有书签的，写上内容之后，小节书签丢失。
![default](https://github.com/note286/xduts/assets/60461837/55965247-dfa4-4b49-93a2-8672599aecb4)

![default2](https://github.com/note286/xduts/assets/60461837/1d4fe882-5e15-4077-a801-dda3e5d4395b)

**Comment by note286** [comment, marked as answer]
请确认使用最新版的xduts，以及进行了两次XeLaTeX编译。

**Reply by during123** [reply]
好的，感谢大佬！！！
